git_machete/tests/functional/test_machete.py

```diff
@@ -1382,3 +1531,239 @@ def test_update_with_fork_point_specified(self) -> None:
                 "<commit_hash>' drops the commits until (included) fork point "
                 "specified by the option '-f' from the current branch."
         )
+
+    @mock.patch('urllib.request.urlopen', ContextManager)
+    @mock.patch('urllib.request.Request', GitAPIState)
```

Comment:
Hmmm since `GitAPIState` is a mock for **Request**... should it be called `GitAPIState`? Shouldn't it rather be called `MockGithubAPIRequest` instead?